app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
+
 .check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+  .set {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
```

Comment:
Might be worth holding off until there's some form of consensus from the new GOV.UK Frontend work.